src/test/test_utils.h

```diff
@@ -56,6 +61,27 @@ inline std::filesystem::path createDataForOnlineDatabase() {
 inline void clearDataForOnlineDatabase(const std::filesystem::path &path) {
   std::filesystem::remove_all(path);
 }
+
+inline image_sequence_localizer::CostMatrix
+computeCostMatrix(const fs::path &queryDir, const fs::path &refDir) {
+
+  std::vector<std::string> queryFiles = listProtoDir(queryDir, ".Feature");
+  std::vector<std::string> refFiles = listProtoDir(refDir, ".Feature");
+  image_sequence_localizer::CostMatrix cost_matrix;
+
+  for (const auto &query : queryFiles) {
+    auto queryFeature = CnnFeature(query);
+    for (const auto &ref : refFiles) {
+      auto refFeature = CnnFeature(ref);
```

Comment:
nit: can be moved to the next line avoiding creating a copy. Not super important here but still a good practice.

Reply:
Didn't get that.

Reply:
What I meant is that you can replace this and the following line with:
```c++
cost_matrix.add_values(queryFeature.computeSimilarityScore(CnnFeature(ref)));
```